internal/core/command/config/config.go

```diff
@@ -34,6 +35,29 @@ type WritableInfo struct {
 	InsecureSecrets bootstrapConfig.InsecureSecrets
 }
 
+type MessageQueue struct {
+	Internal InternalMessageQueue
+	External ExternalMQTT
+}
+
+type InternalMessageQueue struct {
+	bootstrapConfig.MessageBusInfo
+	RequestTopicPrefix                 string
+	ResponseTopic                      string
+	InternalRequestCommandTopic        string
+	InternalResponseCommandTopicPrefix string
+	InternalRequestQueryTopic          string
+	InternalResponseQueryTopic         string
+}
+
+type ExternalMQTT struct {
+	bootstrapConfig.MessageBusInfo
+	RequestCommandTopic        string
+	ResponseCommandTopicPrefix string
+	RequestQueryTopic          string
+	ResponseQueryTopic         string
+}
+
```

Comment:
`GetBootstrap()` below needs set MessageQueue property like is done here so the existing Messaging Handler can get the data needed.
 https://github.com/edgexfoundry/edgex-go/blob/main/internal/core/data/config/config.go#L80